# name: bdb.Bdb.user_excepti 
bdb.Bdb.user_exception